policy/reviewer/reviewer.go

```diff
@@ -124,31 +127,94 @@ func FindRandomRequesters(ctx context.Context, prctx pull.Context, result common
 			shoveIntoMap(allUsers, orgMembers)
 		}
 
-		allCollaboratorPermissions, err := prctx.RepositoryCollaborators()
+		allCollaboratorPermissions := make(map[string]string)
+		allCollaboratorAndPermissions, err := prctx.RepositoryCollaborators()
```

Comment:
The distinction between these variables is unclear... what does the extra `And` in the second one mean for the functionality?

Reply:
I was trying to come up with a better name here. The idea is that we want to only consider some collaborators, and not all, when checking which admins can be auto-assigned. I've tried to rename these to `allCollaborators` and `collaboratorsToConsider` to be a little more clear.